Tools/cases_generator/analyzer.py

```diff
@@ -483,6 +505,49 @@ def make_uop(name: str, op: parser.InstDef, inputs: list[parser.InputEffect]) ->
         body=op.block.tokens,
         properties=compute_properties(op),
     )
+    if effect_depends_on_oparg_1(op) and "split" in op.annotations:
+        result.properties.oparg_and_1 = True
```

Comment:
For code consistency, shouldn't this be `compute_properties`?

Reply:
Shouldn't what be `compute_properties`?

Reply:
The `oparg_and_1` property only applies to the base uop, not the replicas as their behavior does not depend on `oparg & 1`. `compute_properties` computes the properties from the definition only, so we would need to modify `oparg_and_1` anyway.